tests/extmod/vfs_lfs.py

```diff
@@ -101,6 +101,14 @@ def test(bdev, vfs_class):
     print(vfs.getcwd())
     vfs.chdir("/testdir")
     print(vfs.getcwd())
+
+    # create file in directory to make sure paths are relative
+    vfs.open("test2", "w").close()
+    print(vfs.stat("test2"))
+    print(vfs.stat("/testdir/test2"))
```

Comment:
How about adding a pile of additional prints for: `/foo/../test2`, `/foo/bar/../../test2`, `foo/bar/../../test2`, `../testdir/test2`, `/../../../testdir/test2`, `/testdir/foo/../test2`, these shuold all succeed (if I got them right) and print the stat of the test2 file.

Reply:
Ooops, ignore here, what I wrote would be good to test #5988 and doesn't apply to this PR.